lib/mix/lib/mix/task.ex

```diff
@@ -7,11 +7,11 @@ defmodule Mix.Task do
   in a module starting with `Mix.Tasks.` and defining
   the `run/1` function:
 
-      defmodule Mix.Tasks.Hello do
+      defmodule Mix.Tasks.Echo do
         use Mix.Task
 
-        def run(_args) do
-          Mix.shell.info "hello"
+        def run(args) do
+          Mix.shell.info Enum.join(args, " ")
```

Comment:
It indeed works as the `echo` command. I didn't think we want to mock `echo` verbatim, rather echo the arguments we got. If our goal is to mimic `echo` then 👍.